src/Component/Style/Style.tsx

```diff
@@ -102,14 +100,8 @@ export interface StyleProps extends Partial<StyleDefaultProps> {
   onStyleChange?: (style: GsStyle) => void;
   /** The data projection of example features */
   dataProjection?: string;
```

Comment:
`ruleRendererType`, `iconLibraries`, `colorRamps`, `enableClassification` and `dataProjection` (needs to be added somewhere. probably to the renderer, if it does not exist yet) can also be removed.

Reply:
a21cbdd and e15401c